Candidate/Services/CandidateService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Data.SqlClient;
+using System.Text.Json;
+using Candidate.Models;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+
+namespace Services
+{
+	public class CandidateServices
+	{
+        private readonly IConfiguration _configuration;
+
+        public  CandidateServices(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public Candidate.Models.Candidate GetCandidate(int id)
+        {
+            try
+            {
+                Candidate.Models.Candidate candidate = null;
+
+                using (SqlConnection connection = new SqlConnection(_configuration.GetConnectionString("DefaultConnection")))
```

Comment:
Consider creating a central place to get the SqlConnection ... this way if the connection string name changes, its a single change, not a hundred.

Reply:
Great Idea

Reply:
Added